Server/Packets/CryptoContext.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography;
+using System.Diagnostics;
+using Serilog;
+
+namespace Moonlapse.Server.Packets {
+    public class CryptoContext {
+        // The RSA keys should go in the application base directory
+        readonly static string keysPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Keys");
+
+        private byte[]? clientAESPrivateKey;
```

Comment:
```suggestion
        byte[]? clientAESPrivateKey;
```

`private` is redundant